src/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import registerServiceWorker from './registerServiceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
Pls, create `redux-store` and integrate app with it

Reply:
Or you will do this in next PR?

Reply:
I am going to add it in next PR, if it is ok for you. Or it's better to do it in scope of this PR?